src/core/gui/inputdevices/PenInputHandler.cpp

```diff
@@ -5,6 +5,7 @@
 #include "PenInputHandler.h"
 
 #include <algorithm>  // for max, min
+#include <cassert>    // for assert
```

Comment:
```suggestion
#include "util/Assert.h"  // for xoj_assert
```
(+ associated clang-formatting)